src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1036,4 +1036,8 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>You are running the '{0}' operation with an 'http' source, '{1}'. Support for 'http' sources will be removed in a future version.</value>
     <comment>0 - The command name. Ex. Push/Delete. 1 - server uri.</comment>
   </data>
+  <data name="Warning_HttpSource" xml:space="preserve">
+    <value>A 'http' source, '{0}', was found. Support for 'http' sources will be removed in a future version.</value>
```

Comment:
I think we are just trying to avoid using HTTP, as Jeff said https://github.com/NuGet/NuGet.Client/pull/4600#discussion_r863066066, there can be more schemas other than HTTPS. I tried to stay close to a similar message we have: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.PackageManagement/Strings.resx#L432

Reply:
The string you linked is fine, why not use it? I like the one you linked better than the one in this PR for the reasons I specified above about being more declarative.